app/frontend/components/pages/delegations/delegationHistoryPage.tsx

```diff
@@ -0,0 +1,145 @@
+import {h, Component, Fragment} from 'preact'
+import actions from '../../../actions'
+import {connect} from '../../../libs/unistore/preact'
+import {CopyPoolId} from './common'
+import toLocalDate from '../../../../frontend/helpers/toLocalDate'
+
+export enum DelegetionHistoryItemType {
+  StakeDelegation,
+  StakingReward,
+  RewardWithdrawal,
+}
+
+export interface DelegetionHistoryObject {
+  type: DelegetionHistoryItemType
+  epoch: number
+  dateTime: Date
+}
+
+export interface StakePool {
+  id: string
+  name: string
+}
+
+export interface StakeDelegation extends DelegetionHistoryObject {
+  newStakePool: StakePool
+  oldStakePool?: StakePool
+}
+
+const EpochDateTime = ({epoch, dateTime}: {epoch: number; dateTime: Date}) => {
+  return (
+    <div className="grey margin-bottom">
+      Epoch {epoch}, {toLocalDate(dateTime)}
+    </div>
+  )
+}
+
+const StakeDelegationItem = ({stakeDelegation}: {stakeDelegation: StakeDelegation}) => {
+  return (
+    <li className="delegations-history-item">
+      <div className="label">Stake Delegation</div>
+      <EpochDateTime epoch={stakeDelegation.epoch} dateTime={stakeDelegation.dateTime} />
+      <div>
+        New pool: <span className="bold">{stakeDelegation.newStakePool.name}</span>
+        <CopyPoolId value={stakeDelegation.newStakePool.id} />
+      </div>
+      {stakeDelegation.oldStakePool ? (
+        <div>
+          Previous pool: {stakeDelegation.oldStakePool.name}
+          <CopyPoolId value={stakeDelegation.oldStakePool.id} />
+        </div>
+      ) : (
+        ''
+      )}
+    </li>
+  )
+}
+
+export interface StakingReward extends DelegetionHistoryObject {
+  reward: number
```

Comment:
this is the amount in Lovelaces, check the type _Lovelace_ in _app/frontend/state.ts_

Reply:
Fixed